llvm/lib/Target/Sparc/SparcSubtarget.h

```diff
@@ -45,7 +45,8 @@ class SparcSubtarget : public SparcGenSubtargetInfo {
 
 public:
   SparcSubtarget(const Triple &TT, const std::string &CPU,
-                 const std::string &FS, const TargetMachine &TM, bool is64bit);
+                 const std::string &TuneCPU, const std::string &FS,
```

Comment:
While here, could change all `const std::string &` to `StringRef` and/or remove the `TT` parameter (it can be obtained as `TM.getTargetTriple()`.).